torch_tvm/version.py

```diff
@@ -0,0 +1,9 @@
+# This file is generated by setup.py. DO NOT EDIT!
```

Comment:
what's this used for? can we add it to gitignore instead?

Reply:
ah this shouldn't even be in this commit